### DISCUSSION TITLE
✅ Icon list element transition on hover - [ED-9742]

### DISCUSSION THREAD

**Discussion by wayheming** [elementor/elementor#15479] [Feature Request]
## Description

Hello, in element Icon list there is no option to select the transition and there is no standard transition and as you can see on the gif hover is not very beautiful.

![a3c90ca5d60524146d7701abffd24fc3](https://user-images.githubusercontent.com/31981892/55955895-4d523500-5c6b-11e9-93f7-c2f10a4c08ab.gif)

**Comment by wayheming** [comment]
@shilo-ey @bainternet Hello, сan anyone looks at the problem? Thanks

**Comment by wayheming** [comment]
@shilo-ey Hello, you said that consider the option of making my pull request. Soon 2.6 you can add it to this version? #7868

**Comment by nicholaszein** [comment]
Hello there @XXX!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor **v3.12.1**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
